src/lib/capacityEngine.js

```diff
@@ -77,9 +120,7 @@ function capacityEngine(store, productIds, time, period = 30) {
         // the maximum price a user would get can only be determined if the entire available
         // capacity is bought because the routing will always pick the cheapest
         // so we're summing up the premium for all pools and then calculate the average at the end
-        const poolPremium = product.useFixedPrice
-          ? calculateFixedPricePremiumPerYear(availableInNXM, basePrice)
-          : calculatePremiumPerYear(availableInNXM, basePrice, usedInNxm, totalInNXM);
+        const poolPremium = calculateMaxPremium(product, pool, basePrice, time);
```

Comment:
fixed price products don't have surge, so it's fine to not enter to for loop and return here, as before